.github/workflows/test-internal-prs.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 
```

Comment:
Can we change the name of the file to add `poc-` prefix, and also change the `name` field above? Sometimes GHA has trouble deduping different workflows if their names are the same.